modules/videopress/videopress.php

```diff
@@ -731,6 +759,33 @@ function videopress_shortcode_options( $options ) {
 		return $options;
 	}
 
+	/**
+	 * Get the upload api path.
+	 *
+	 * @param int $blog_id The id of the blog we're uploading to.
+	 * @return string
+	 */
+	protected function make_media_upload_path( $blog_id ) {
+		return sprintf(
+			'%s://%s/rest/v%s/videos/%s/new',
+			'https',
```

Comment:
If it were a ternary or the like that toggled between http and https, I'd maybe understand it, but if it's always going to be https, I'd sooner see that just replacing the %s in the original string instead of making php do that work on every load.